recipes/typechecks/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: typechecks
+  version: "0.1.0"
+
+source:
+  fn: typechecks-0.1.0.tar.gz
+  url: https://files.pythonhosted.org/packages/62/21/15129201c1f52f6af1e7809e96dce46da4b406c2c07fe9425e92f51edc5c/typechecks-0.1.0.tar.gz
+  md5: 140956d8f5991b9b73769431b95aeff8
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - typechecks
+
+about:
+  home: https://github.com/openvax/typechecks
+  license: Apache
```

Comment:
`license: Apache-2.0`

Also please manually include the `license_file`:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually